public/css/style.css

```diff
@@ -17,26 +19,39 @@
 
 .thumb {
     margin: 1px 1px -3px 1px;
-    padding: 0px 0px 0px 0px;
+    padding: 0 0 0 0;
     width: 150px;
     height: 150px;
 }
 
+/* Centered colums for grid display in Nebula */
 .col-centered {
-    display:inline-block;
-    float:none;
-    text-align:left;
-    margin-right:-4px;
+    display: inline-block;
+    float: none;
+    text-align: left;
+    margin-right: -4px;
 }
 
+/* Centered rows for grid display in Nebula */
 .row-centered {
-    text-align:center;
+    text-align :center;
 }
 
+/* Plot all D3 SVGs to center of their divs */
 .plotbox {
     text-align: center;
 }
 
+/* GENERAL PLOT CSS */
+
+.axis path, .axis line {
+    fill: none;
+    stroke: black;
+    shape-rendering: crispEdges;
```

Comment:
Haha this is great value. I think we might want to go with superCrisp though. =P